### ISSUE TITLE
unintended behavior with showif

### ISSUE THREAD

**Issue by Elckarow** [renpy/renpy#5268]
```rpy
screen test():
    default var = False

    text str(var) align (1.0, 0.0)

    showif var:
        add "#000":
            at transform:
                xysize (30, 30) align (0.5, 0.5)
                on show:
                    alpha 0.0 zoom 1.0
                    linear 0.5 alpha 1.0
                on hide:
                    zoom 1.0
                    linear 0.5 alpha 0.0 zoom 2.0
    
    key "K_a" action ToggleScreenVariable("var")
    key "K_b" action (Hide("test"), With(Dissolve(0.5)))
```

hiding the screen when `var` is true will trigger the `showif`'s hide event, which shouldn't happen

**Comment by mal** [comment]
Best guess is that it's related to e19a29f20afcac62fac0e1ff0f4267d74d1fa136.
Does `config.containers_pass_transform_events = False` prevent this behaviour?

**Comment by Elckarow** [comment]
it does
and btw, the doc says `containers_pass_transform_events = False` instead of `config.containers_pass_transform_events = False`

**Comment by mal** [comment]
This should be resolved in `8.2.0.24011603+nightly`.

**Comment by Elckarow** [comment]
working as intended 👍
